spec/doubleLinkedList_test.js

```diff
@@ -0,0 +1,156 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoubleLinkedList from '../src/doubleLinkedList'
+
+chai.use(chaiChange)
+
+describe('Double Linked List', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(DoubleLinkedList).to.be.a('function')
+  })
+
+  describe('insert()', () => {
+    it('Inserts a node (with the provided data) to the tail of the list', () => {
+      const doubleLinkedList = new DoubleLinkedList()
+
+      expect(() => doubleLinkedList.insert('foo'))
+        .to.alter(() => doubleLinkedList.count, { from: 0, to: 1 })
+      expect(doubleLinkedList.getHeadNode().data)
```

Comment:
Instead of relying on the internal data representation in `Node`, consider adding an accessor method:
```js
doubleLinkedList.getHeadNode().getData()
```
If you later choose to change how data is represented in the node, since you have *programmed to the interface, instead of the implementation*, your code (both in this test and in the list implementation) will not need to change.